common/men_at_arms_types/00_regional_maa_types.txt

```diff
@@ -1 +1,70 @@
-﻿# Warcraft
\ No newline at end of file
+﻿# standard costs
+@maa_buy_cost = 150
+@maa_low_maintenance_cost = 1.0
+@maa_high_maintenance_cost = 5.0
+#Warcraft
+war_camel_rider = {
```

Comment:
Keep the vanilla name, it's easier to handle.

Reply:
the localization wasn't being overridden when I kept the vanilla name

Reply:
> the localization wasn't being overridden when I kept the vanilla name

Do you remember you need to place it in the replace folder?